cypress/e2e/viewOrgBounties.cy.ts

```diff
@@ -0,0 +1,52 @@
+describe('View Organization Bounties', () => {
+  const org: Cypress.Organization = {
+    loggedInAs: 'alice',
+    name: 'Organization',
```

Comment:
@chiragDork Change the name of the Organization to `OrganizationView` or something else other than Organization, it failed on my system because I have a wider screen, and it can capture multiple Organizations on the Screen.

Make the Organization name a variable, so you don't have to repeat yourself.